offline/cable_cru_TRENDY.F90

```diff
@@ -451,18 +636,43 @@ SUBROUTINE cru_get_daily_met(CRU, LastDayOfYear)
     DummyYear = CRU%cYear + 1
   END IF
 
-  ! Was the Tmin recycled?
+  ! Now we have to handle each of the "next" variables individually
+  ! Tmax
   IF (CRU%isRecycled(Tmin)) THEN
-    DummyYear = CRU%RecycStartYear + MOD(DummyYear - 1501, CRU%metRecyc)
+    DummyYear = CRU%RecycStartYear + MOD(DummyYear - 1501, CRU%MetRecycPeriod)
   END IF
 
   CALL read_metvals(CRU%MetDatasets(Tmin), CRU%Met(nextTmin)%MetVals, land_x,&
     land_y, DummyYear, DummyDay, CRU%LeapYears, CRU%xDimSize, CRU%yDimSize,&
     CRU%DirectRead)
 
-END SUBROUTINE cru_get_daily_met
+  ! vph1500
```

Comment:
The code afterwards is using vph0900 everywhere, so why does the comment say "vph1500"?